phases/05-nlp-foundations-to-advanced/09-sequence-to-sequence/code/main.py

```
@@ -0,0 +1,61 @@
+import math
+import random
+
+
+def simulate_copy_accuracy(seq_len, context_dim=8, epochs=200, n_train=300, seed=0):
+    rng = random.Random(seed)
```

Ruff (0.15.10)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)